ConnectSecure/Deploy-ConnectSecure-V4.ps1

```diff
@@ -61,8 +61,9 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #>
 [CmdletBinding()]
 param(
-    [Parameter(Mandatory=$false)][int] $CompanyID = $env:companyid,
+    [Parameter(Mandatory=$false)][int] $CompanyID = (Ninja-Property-Get configureconnectsecurecompanyid),
```

Comment:
Would like to keep this set to $env:companyid in order to keep Script Variables working, and then check for "configureconnectsecurecompanyid" field with Ninja-Property-Get existing later and use it if the Script Variable (environment variable) is blank as an alternate. This would let the Script Variable override if provided but be optional if the custom field exists.

Reply:
Reverted, and added logic to check Ninja custom field if Ninja doc doesn't exist.